pkgs/by-name/op/opensta/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  tcl,
+  swig,
+  bison,
+  flex,
+  git,
+  cudd,
+  eigen,
+  libzip ? null,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "opensta";
+  version = "0-unstable-2025-04-24";
+
+  src = fetchFromGitHub {
+    owner = "parallaxsw";
+    repo = "OpenSTA";
+    rev = "cba0cf5cc43f5ba484bfe8e80f137a6d71dff56e";
+    sha256 = "1v8xicv5syp3x6gn5f2rdn9rdg9pj6x60qdkdik8izzc25s2cjac";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+    git
+    eigen
+    cudd
+    swig
+  ];
+
+  buildInputs = lib.optional stdenv.isLinux libzip ++ [ tcl ];
+
+  meta = with lib; {
+    description = "A gate‑level static timing verifier";
+    homepage = "https://github.com/parallaxsw/OpenSTA";
+    license = licenses.gpl3Plus;
+    maintainers = [ maintainers.fayalalebrun ];
+    platforms = platforms.unix;
+  };
+}
```

Use of `with lib;` is generally frowned upon these days.

```suggestion
  meta = {
    description = "A gate‑level static timing verifier";
    homepage = "https://github.com/parallaxsw/OpenSTA";
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [ fayalalebrun ];
    platforms = lib.platforms.unix;
  };
}
```